backend/app/tests/services/doctransformer/test_zerox_transformer.py

```
@@ -0,0 +1,438 @@
+from unittest.mock import Mock, patch
+
+import pytest
+
+from app.services.doctransform.zerox_transformer import ZeroxTransformer
+from app.services.doctransform.transformer import Transformer
+
+
+def create_async_mock(return_value):
+    """Helper to create an async function that returns a value."""
+
+    async def async_func(*args, **kwargs):
```

Ruff (0.14.8)

Unused function argument: `args`

(ARG001)